js/main.js

```diff
@@ -0,0 +1,22 @@
+const submit = document.querySelector('#submit');
+const form = document.querySelector('form');
+const navIcon = document.querySelector('#nav-icon');
+const nav = document.querySelector('nav');
+
+form.addEventListener('submit', (e)=> {
+  e.preventDefault();
+  alert('We Have Received Your Message!');
+  form.reset();
+
+});
+
+//nav functionality
+navIcon.addEventListener('click',()=> {
+  navIcon.classList.toggle("change");
+
+  nav.classList.toggle('show');
```

Comment:
Looks like a way for you to add button functionality for displaying a nav menu from the side - nice touch! Would be great to finish working on this when you have time.